app/models/physical_server_profile.rb

```diff
@@ -0,0 +1,29 @@
+class PhysicalServerProfile < ApplicationRecord
+  acts_as_miq_taggable
+
+  include NewWithTypeStiMixin
+  include TenantIdentityMixin
+  include SupportsFeatureMixin
+  include EventMixin
+  include ProviderObjectMixin
+  include EmsRefreshMixin
+
+  include_concern 'Operations'
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :inverse_of => :physical_server_profiles,
+    :class_name => "ManageIQ::Providers::PhysicalInfraManager"
+
+  belongs_to :assigned_server, :optional => true, :class_name => "ManageIQ::Providers::PhysicalInfraManager::PhysicalServer"
```

Comment:
@agrare Can you help me with these relations?

Here, we get a one-directional relation so that a `server_profile` has `assigned_server_id`. There are two problems that I have with this and should probably be fixed.

First, the `:class_name` should probablz be `::PhysicalServer`. Only in this case, I get proper server using `profile.assigned_server`.

Second, how would I achieve that an instance of `PhysicalServer` gets attribute `assigned_profile` (and also `associated_profile` using relations? I think there should be something with `inverse_of` but I just can't seem to find a proper relationship.

Reply:
Solved it.